src/main/java/org/killbill/billing/plugin/adyen/api/AdyenPaymentPluginApi.java

```diff
@@ -158,8 +160,8 @@ public AdyenPaymentPluginApi(final AdyenConfigurationHandler adyenConfigurationH
         this.adyenHppConfigurationHandler = adyenHppConfigurationHandler;
         this.dao = dao;
 
-        final KillbillAdyenNotificationHandler adyenNotificationHandler = new KillbillAdyenNotificationHandler(killbillApi, dao, clock);
-        this.adyenNotificationService = new AdyenNotificationService(ImmutableList.<AdyenNotificationHandler>of(adyenNotificationHandler));
+        final AdyenNotificationHandler adyenNotificationHandler = new KillbillAdyenNotificationHandler(killbillApi, dao, clock);
```

Comment:
ok. Btw. the plugin does not compile with 1.6 at the moment. Also the travis build only checks with 1.7+